PlaceRouteHierFlow/router/GcellDetailRouter.cpp

```diff
@@ -4457,7 +4448,10 @@ void GcellDetailRouter::ConvertRect2GridPoints_Via_set(std::vector<std::set<Rout
           if (x >= LLx && x <= URx && y >= LLy && y <= URy) {
             tmpP.x = x;
             tmpP.y = y;
-            plist.at(mIdx).insert(tmpP);
+            if (up)
+              grid.vertices_total[grid.vertices_total_map.at(mIdx)[tmpP]].via_active_up = false;
+            else
+              grid.vertices_total[grid.vertices_total_map.at(mIdx)[tmpP]].via_active_down = false;
```

Comment:
inactivate vertice in place instead of storing point into set

Reply:
Elapsed Time:	263.661s
    CPU Time:	261.265s
    Total Thread Count:	66
    Paused Time:	0s


![1669224714377](https://user-images.githubusercontent.com/28666144/203611867-e045ca5f-c0df-478b-ad67-1704980d0f7a.png)